plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/AppEngineLibraryContainerInitializerTest.java

```diff
@@ -61,95 +72,96 @@
-    setupLibraryBuilder();
+    setupLibraryFactory();
 
     AppEngineLibraryContainerInitializer containerInitializer =
         new AppEngineLibraryContainerInitializer(new IConfigurationElement[]{ configurationElement },
-                                                 libraryBuilder,
+                                                 libraryFactory,
                                                  TEST_CONTAINER_PATH);
     containerInitializer.initialize(new Path("single.segment.id"),
                                     testProject.getJavaProject());
   }
 
   @Test(expected = CoreException.class)
   public void testInitialize_containerPathConsistsOfThreeSegments() throws Exception {
-    setupLibraryBuilder();
+    setupLibraryFactory();
 
     AppEngineLibraryContainerInitializer containerInitializer =
         new AppEngineLibraryContainerInitializer(new IConfigurationElement[]{ configurationElement },
-                                                 libraryBuilder,
+                                                 libraryFactory,
                                                  TEST_CONTAINER_PATH);
     containerInitializer.initialize(new Path("first.segment/second.segment/third.segment"),
                                     testProject.getJavaProject());
   }
 
   @Test(expected = CoreException.class)
   public void testInitialize_containerPathHasWrongFirstSegment() throws Exception {
-    setupLibraryBuilder();
+    setupLibraryFactory();
 
     AppEngineLibraryContainerInitializer containerInitializer =
         new AppEngineLibraryContainerInitializer(new IConfigurationElement[]{ configurationElement },
-                                                 libraryBuilder,
+                                                 libraryFactory,
                                                  TEST_CONTAINER_PATH);
     containerInitializer.initialize(new Path("first.segment/second.segment"),
                                     testProject.getJavaProject());
   }
 
   @Test(expected = CoreException.class)
   public void testInitialize_containerPathHasWrongLibraryId() throws Exception {
-    setupLibraryBuilder();
+    setupLibraryFactory();
 
     AppEngineLibraryContainerInitializer containerInitializer =
         new AppEngineLibraryContainerInitializer(new IConfigurationElement[]{ configurationElement },
-                                                 libraryBuilder,
+                                                 libraryFactory,
                                                  TEST_CONTAINER_PATH);
     containerInitializer.initialize(new Path(TEST_CONTAINER_PATH + "/second.segment"),
                                     testProject.getJavaProject());
   }
 
   @Test
-  public void testInitialize_libraryBuilderErrorDoesNotPreventOtherLibraries() throws Exception {
+  public void testInitialize_libraryFactoryErrorDoesNotPreventOtherLibraries() throws Exception {
```

Comment:
Would it make sense to move this to a separate test class so that `setupLibraryFactory()` could be merged into your `@Before`/`setUp()`?